CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ the code was deployed.
 
 ## [Unreleased]
 
+## Added
+
+- PA route /pa/check-database-connection to check if database connection is working as expected (CU-86dqkw7ef).
```

Comment:
@micahdbak , when you make the next change in Sensors, would you please update the route in the CHANGELOG to `/pa/health`? Thanks!